.github/workflows/android-release-to-org-openfoodfacts-scanner.yml

```diff
@@ -90,6 +93,16 @@ jobs:
          SIGN_KEY_ALIAS: ${{ secrets.SIGN_KEY_ALIAS }}
          SIGN_KEY_PASSWORD: ${{ secrets.SIGN_KEY_PASSWORD }}
 
+      - name: Upload binaries to release
+        uses: svenstaro/upload-release-action@v2
+        if: github.event.inputs.RELEASE_TAG != ''
+        continue-on-error: true
+        with:
+          file: ./packages/smooth_app/build/app/outputs/bundle/release/app-release.aab
+          asset_name: android-release-${{ inputs.RELEASE_TAG }}.apk
```

Comment:
is it ok to just rename the AAB @M123-dev @g123k ?

Reply:
No it's not. We should build an APK with flutter build and then upload it.

Reply:
Ohh yes right, that was not intended, I just copied the path from the Fastfile, and typed a .apk name without even thinking about that it's a apk.


But that's an interesting topic, should we just provide a single apk or multiple architechtures

Reply:
The apk generated should be good for every architecture. About APK size i don't think we should care much as it is a fallback method (we have PlayStore and in the future fdroid)

Reply:
Just checking: we still create an AAB (e.g. for Play Store), _and_ an APK (e.g. for Amazon), right?

Reply:
We don't currently auto-deploy to Amazon. They still require APKs

Reply:
This should be fixed now, I have extracted the github release to a own run, where we build a fat apk which should be uploaded to the github releases.

@monsieurtanuki yes we still build an AAB for the playstore